src/pages/Cart/Cart.scss

```
@@ -0,0 +1,116 @@
+.cart {
+  width: 70%;
+  height: 400px;
+  margin: auto;
+
+  .cartHeader {
+    padding-bottom: 10px;
+    text-align: center;
+  }
+
+  .cartContainer {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    padding: 10px 0;
+    border-top: 1px solid black;
+  }
+
+  .cartProduct {
+    width: 50%;
+    margin: 20px;
+    padding-right: 20px;
+
+    .cartProductOption {
+      display: flex;
+      justify-content: space-between;
+      width: 95%;
+
+      .cartProductDeleteBtn {
+        border: none;
+        background-color: #ffbb0f;
+      }
+    }
+    .cartProductList {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      width: 90%;
+      margin: 20px 0;
+
+      .cartProductItem {
+        display: flex;
+        align-items: center;
+        justify-content: space-around;
+        padding: 10px 0;
+        border-top: 1px solid black;
+      }
+      .cartProductItem:last-child {
+        border-bottom: 1px solid black;
+      }
+    }
+
+    .cartProductItem {
+      .cartProductItemImg {
+        width: 50px;
+        height: 50px;
+      }
+
+      .cartProductName {
+        width: 300px;
+        text-align: center;
+      }
+      .cartProductPrice {
+        width: 61px;
+        text-align: center;
+      }
+    }
+  }
+
+  .cartPurchase {
+    align-self: flex-start;
+    margin: 20px auto 0 auto;
+
+    .cartPurchaseKey {
+      margin: 10px 0;
+
+      &:nth-child(2) {
+        padding-bottom: 10px;
+        border-bottom: 1px solid black;
+      }
+
+      &:nth-child(3) {
+        color: #ffbb0f;
+      }
+
+      .keyValue {
+        margin-left: 5px;
+      }
+    }
+    .cartPurchaseBtn {
+      width: 100%;
+      margin: 10px 0;
+      border: none;
+      background-color: #ffbb0f;
+    }
+    .deliveryAlert {
+      position: relative;
+      .deliveryAlertMessage {
+        height: 15px;
```

```suggestion
    .deliveryAlert {
      position: relative;
      
      .deliveryAlertMessage {
        height: 15px;
```
스타일끼리는 가독성을 위해 한칸씩 띄워주세요~